sklearn/cluster/k_means_.py

```diff
@@ -684,6 +684,16 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
         given, it fixes the seed. Defaults to the global numpy random
         number generator.
 
+    verbose : boolean, optional
+        Verbosity mode.
+
+    copy_x : boolean, optional
+        When pre-computing distances it is more numerically accurate to center
+        the data first.  If copy_x is True, then the original data is not
```

Comment:
Done!

Also the verbose default was 0... I changed to False for consistency with other modules...